apps/condo/codegen.ts

```diff
@@ -42,17 +42,17 @@ const config: CodegenConfig = {
         'gql/index.ts': {
             preset: 'import-types',
             presetConfig: {
-                typesPath: '@/gql/operation.types',
+                typesPath: '@app/condo/gql/operation.types',
             },
             plugins: [
                 codegenCommentPlugin(),
                 'typescript-react-apollo',
-                exportFromPlugin('@/gql/operation.types'),
+                exportFromPlugin('@app/condo/gql/operation.types'),
             ],
         },
     },
     hooks: {
-        afterAllFileWrite: ['eslint --fix --no-ignore'],
+        afterAllFileWrite: ['eslint --fix ./gql/operation.types.ts ./gql/index.ts ./schema.ts'],
```

Comment:
why do you need to specify files here?

Reply:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/783e4c44-1b1d-466e-9d59-2ea8b9e796ac">

Reply:
we need to specify the files, otherwise eslint will process every file, not just the generated ones

Reply:
This greatly speeds up the execution of the script

Reply:
I was wrong.. This doesn't help much. It seems you shouldn't run eslint on codegen

Reply:
Our generated files are very large. And because of this, the script execution takes a huge amount of time.